### ISSUE TITLE
aws_eip resource missing support for tags in AWS provider v1.6.0

### ISSUE THREAD

**Issue by whereisaaron** [terraform-providers/terraform-provider-aws#2805]
### Terraform and AWS Provider Versions

```
Terraform v0.11.1
+ provider.aws v1.6.0
```

### Terraform Configuration Files

```hcl
resource "aws_eip" "example" {
  vpc      = true
  instance = "i1234567890abcdef"

  tags {
    Name = "EIP for my favourite instance
  }
}
```

### Expected Behavior

EIP created and and tags added

### Actual Behavior

Error: aws_eip.example: : invalid or unknown key: tags

### Steps to Reproduce

1. `terraform init`
2. `terraform apply`

### Important Factoids

Tested in us-east-2 region

### References

Couldn't find any existing issue covering the lack of support for tags on EIP resources. [Manual page for aws_eip resource](https://www.terraform.io/docs/providers/aws/r/eip.html) supports finding that 'tags' is not supported. @arnoldokoth over in the terraform project [pointed out they where added to AWS only 9 days ago](https://github.com/hashicorp/terraform/issues/16993#issuecomment-354549732) so no surprise the provider doesn't have them yet! But hopefully easy to add if uniform with other resource tags.

**Comment by jen20** [comment]
Hi @whereisaaron! Support was merged to the provider in #2768, however there has not been a release yet as HashiCorp is on their Christmas break. I have no doubt there will be a release shortly after the new year which will include this!

**Comment by whereisaaron** [comment]
Sweet! Thanks @jen20.
